pallets/marketplace/src/benchmarking.rs

```diff
@@ -44,9 +41,8 @@ where
     > = Default::default();
     let r = pricing.try_push(PricingVariant {
         reward_asset: 0.into(),
-        price_per_cpu_millisecond: price_per_cpu_millisecond.into(),
-        bonus: 0.into(),
-        maximum_slash: 0.into(),
+        fee_per_millisecond: price_per_millisecond.into(),
+        fee_per_execution: 0.into(),
```

Comment:
Just so I understand. The `fee_per_execution` is the minimum cost and `fee_per_millisecond` is an extra that can be provided?

Reply:
exactly, fix base fee (can be 0) and fee per millisecond (can be 0)